personalpalate/orm/model.py

```diff
@@ -1,13 +1,46 @@
+from datetime import date
 from typing import Optional
 
 from sqlmodel import SQLModel, Field
 from sqlalchemy import PrimaryKeyConstraint
 
 
-# TODO add model
-class AccountDTO(SQLModel):
+class AccountDTO(SQLModel, table=True):
     email: str = Field(primary_key=True)
+    followable: bool
 
-
-class Account(AccountDTO, table=True):  # Subclass to avoid sending password to client
+class Account(AccountDTO):  # Subclass to avoid sending password to client
     password: str
+
+class Follow(SQLModel, table=True):
+    email: str = Field(default=None, foreign_key="account.email")
```

Comment:
These should not default None. Default None will cause a key to be automatically created, which is only useful for ints and the like. In this case, the email should always be specified